ShellScript/Bash.sublime-syntax

```diff
@@ -3142,7 +3142,7 @@ variables:
 
   # 4.2 Shell Built-in Commands
   # https://www.gnu.org/software/bash/manual/bash.html#Bash-Builtins
-  # Note: Contains thosw, which are not present as dedicated context
+  # Note: Contains items which are not present as dedicated contexts
```

Comment:
the obvious change was `thosw` -> `those` but that didn't help the line make sense...